src/ontology/cl-edit.owl

```diff
@@ -33012,6 +33013,15 @@ SubClassOf(obo:CL_4047032 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0001816))
 SubClassOf(obo:CL_4047032 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0032940))
 SubClassOf(obo:CL_4047032 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0045765))
 
+# Class: obo:CL_4047034 (smooth muscle cell of stomach)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:21443757") Annotation(oboInOwl:hasDbXref "PMID:33630273") obo:IAO_0000115 obo:CL_4047034 "A smooth muscle cell that is part of the muscularis externa of the stomach wall. It is characterized by its fusiform shape, involuntary control, and ability to generate slow, sustained contractions. This cell is organized in distinct layers and is essential for gastric motility and digestion.")
+AnnotationAssertion(terms:contributor obo:CL_4047034 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047034 "2024-12-05T11:10:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047034 "smooth muscle cell of stomach"@en)
+SubClassOf(obo:CL_4047034 obo:CL_0000192)
+SubClassOf(obo:CL_4047034 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004222))
```

Comment:
would suggest adding [stomach muscularis externa](http://purl.obolibrary.org/obo/UBERON_0008856) instead of [stomach smooth muscle](http://purl.obolibrary.org/obo/UBERON_0004222)